### ISSUE TITLE
nuxeo.nuxeo validation results

### ISSUE THREAD

**Issue by christinklez** [ucldc/rikolti#705]

**Comment by barbarahui** [comment]
I'm not seeing any problem for collection 26142 re mapping creator. Can you re-run the validate_by_mapper DAG for it?

**Comment by christinklez** [comment]
@barbarahui you're right! I'm going to strike the "creator" mapping one from the list. Thanks!

**Comment by barbarahui** [comment]
@christinklez re the extra blank space appearing before dates: this is how the data is actually entered in Nuxeo. I've added whitespace trimming to the Nuxeo mapper, but if this is widespread then there was some sort of data entry issue...

**Comment by barbarahui** [comment]
Re `thumbnail_source` for complex objects, we need to replicate this logic, which grabs a thumbnail from a component object if it's available:

https://github.com/ucldc/harvester/blob/master/harvester/fetcher/nuxeo_fetcher.py#L79-L141

**Comment by barbarahui** [comment]
@amywieliczka I guess the best place for the above logic to be implemented is in the content harvester? I'll create an issue for this piece of work.

**Comment by barbarahui** [comment]
@christinklez the is_shown_by for video example you provide above is actually for a complex object. Same issue as for the audio example.

**Comment by christinklez** [comment]
Thanks so much, @barbarahui! Reviewed a set of collections and the mapping fixes look good. Thanks!
